CODE_REVIEW_17.md

```
@@ -0,0 +1,346 @@
+}
+
+// tui/header.go - updateHeader
+switch cv {
+case viewProjectList:
+    // ...
+// Pas de default case
+}
+```
+
+**Suggestion:** Ajouter des default cases défensifs pour la robustesse future :
+```go
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)